src/PerformanceTests/Common/Scenarios/BaseRunner.cs

```diff
@@ -147,12 +239,14 @@ protected virtual Task Stop()
     }
 
 #if Version5
-    Task CreateOrPurgeQueues()
+    async Task CreateOrPurgeQueues()
     {
         var configuration = CreateConfiguration();
         if (IsPurgingSupported) configuration.PurgeOnStartup(true);
```

Comment:
Not draining can result in poisenous messages from other tests to conflict with the new test.